chromium_src/modules/credentialmanager/navigator_credentials_disabled_api_s.h

```diff
@@ -0,0 +1,27 @@
+#ifndef MODULES_NAVIGATOR_CREDENTIALS_DISABLED_APIS_H_
+#define MODULES_NAVIGATOR_CREDENTIALS_DISABLED_APIS_H_
+
+#include "gin/converter.h"
+#include "net/proxy_resolution/proxy_resolver_v8.h"
+#include "third_party/blink/renderer/bindings/core/v8/script_promise.h"
+#include "third_party/blink/renderer/core/frame/navigator.h"
+#include "third_party/blink/renderer/platform/bindings/v8_per_isolate_data.h"
+#include "third_party/blink/renderer/platform/heap/handle.h"
+#include "third_party/blink/renderer/platform/supplementable.h"
+
+namespace blink {
+  class NavigatorCredentialsDisabledAPIs {
+    public:
+      NavigatorCredentialsDisabledAPIs() {
+      }
+      static CredentialsContainer* credentials(Navigator& navigator) {
```

Comment:
this still doesn't match the implementation in browser-laptop and it this will break some sites. We need to handle `get` and `store` and return a promise that resolves to `false`